vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import edu.asu.diging.vspace.core.data.ImageRepository;
+import edu.asu.diging.vspace.core.model.impl.VSImage;
+
+@Controller
+public class ListImagesController {
+	
+	@Autowired
+	private ImageRepository imageRepo;
+	
+	@RequestMapping("/staff/images/list/{page}")
+	public String listSpaces(@PathVariable String page, Model model) {
+		int pageSize = 2; //need to move to a constant file
+		long checker = imageRepo.count()%pageSize;
+		long totalPages = imageRepo.count()/pageSize;
+		totalPages = checker == 0? totalPages:totalPages+1;
+		Pageable sortedByName = PageRequest.of(Integer.parseInt(page)-1, pageSize, Sort.by("creationDate"));
```

Comment:
don't hardcode the sort field name here. Put it in an enum. Eventually we want to be able to sort by different fields.